packages/waku/src/lib/renderers/dev-worker-impl.ts

```diff
@@ -73,8 +79,15 @@ const handleRender = async (mesg: MessageReq & { type: 'render' }) => {
         isDev: true,
         loadServerFile,
         loadServerModule,
-        resolveClientEntry: (id: string) =>
-          resolveClientEntryForDev(id, rest.config),
+        resolveClientEntry: (file: string) => {
+          let id = resolveClientEntryForDev(file, rest.config);
+          for (const moduleNode of initialModules) {
+            if (moduleNode.file === file) {
+              id = moduleNode.url;
```

Comment:
```suggestion
              id = moduleNode.url;
              break;
```

Reply:
I think you probably have, but have you tried adding console.log here?
```suggestion
              console.log('replacing id', id, moduleNode.url);
              id = moduleNode.url;
              break;
```

Does it make any changes between ubuntu and windows?

Reply:
yea tried, let me try again